src/main/java/core/basesyntax/BuilderTestApp.java

```diff
@@ -1,7 +1,8 @@
 package core.basesyntax;
 
 public class BuilderTestApp {
-  public static void main(String[] args) {
-      // Remove this comment and test your Builder implementation here...
-  }
+    public static void main(String[] args) {
+        Plane plane = new Plane.PlaneBuilder().setColor("red").setModel("boeing").build();
```

Comment:
let's set all the fields through the builder

Reply:
done